styleguide/source/_patterns/02-organisms/topic/topic-rail-right/topic-rail-right.twig

```diff
@@ -0,0 +1,3 @@
+<aside class="topic_rail-right">
```

Comment:
I don't think it's valuable IMO, but I suppose it couldn't hurt either. @rpama if you go this route, just make sure the `.md` explains that the pattern isn't included anywhere and is just to show what an example of a right rail looks like on its own